services/src/main/scala/cromwell/pipeline/service/RunService.scala

```diff
@@ -1,31 +1,32 @@
 package cromwell.pipeline.service
 
-import java.time.Instant
-
 import cromwell.pipeline.datastorage.dao.repository.RunRepository
 import cromwell.pipeline.datastorage.dto._
 import cromwell.pipeline.model.wrapper.{ RunId, UserId }
 
+import java.time.Instant
 import scala.concurrent.{ ExecutionContext, Future }
 
 trait RunService {
 
-  def addRun(runCreateRequest: RunCreateRequest): Future[RunId]
+  def addRun(runCreateRequest: RunCreateRequest, userId: UserId): Future[RunId]
```

Comment:
Why do we have to pass `userId` if `runCreateRequest` contains one? What am I missing? :)

Reply:
Yep. You are right! I`l fix it.